bin/run-db-download.py

```diff
@@ -21,9 +21,12 @@
 BUCKET_NAME = os.getenv("AWS_DB_S3_BUCKET", "bedrock-db-dev")
 REGION_NAME = os.getenv("AWS_DB_REGION", "us-west-2")
 S3_BASE_URL = f"https://s3-{REGION_NAME}.amazonaws.com/{BUCKET_NAME}"
+GCS_BASE_URL = f"https://storage.googleapis.com/{BUCKET_NAME}"
+DOWNLOAD_FROM_GCS = os.getenv("DOWNLOAD_FROM_GCS", 'False').lower() in ('true', '1', 't')
```

Comment:
Our tests don't check `black` formatting on these files, but it would be nice if they passed (wrt the single vs double quoting).

Reply:
Sounds good, this line might be a little over the top too. Open to feedback on how you all like to handle boolean values as env variables.

Reply:
In other parts of the code base we use [everett](https://github.com/willkg/everett). If we used it here this could be written as `DOWNLOAD_FROM_GCS = config("DOWNLOAD_FROM_GCS", parser=bool, default="false")`. But not sure why it isn't already used here.

Reply:
oh cool, thanks @robhudson pushed up a change that pulls that in.

Reply:
Welp, maybe that is why looks to fail the tests:
```bash
Traceback (most recent call last):
  File "/app/./bin/run-db-download.py", line 10, in <module>
    from bedrock.base.config_manager import config
ModuleNotFoundError: No module named 'bedrock'
```